environment.yml

```diff
@@ -17,17 +16,17 @@ dependencies:
   - trajectory_analysis_tools
   - matplotlib
   - seaborn
-  - skan
   - bottleneck
   - ipympl
   - tqdm
   - nb_conda
   - pip:
-    - pubnub<6.4.0
+    - skan
```

Comment:
It was in the conda part of the list originally, but if we don't need it, then we should definitely remove it. Is it part of your code?

Reply:
Oh I see. We do use it in `common_position.py`.

 I missed that it moved from conda to pip. Was there a reason for that then?

Reply:
Yes: I also removed conda-forge, because that installs a version of hdf5 that doesn't work with h5py 2.10. scan was only in conda-forge, so I thought it would be best to move it to pip. 

Is there a better way to do that?

Reply:
Hm, I'm hesitant to remove conda-forge because a lot of the latest packages appear there. Pinning h5py to 2.1 doesn't force the correct version of hdf5?

Reply:
I tried it and the version of hdf5 installed was 1.10.6

Is that the wrong version?

Reply:
@edeno Do you want to go ahead and update the pull request with your version? That probably makes the most sense at this point...